tests/cross/src/test/scala/tests/pc/CompletionArgSuite.scala

```diff
@@ -1147,4 +1147,73 @@ class CompletionArgSuite extends BaseCompletionSuite {
        |""".stripMargin,
     topLines = Some(1),
   )
+
+  check(
+    "second-first",
+    """|object Main {
+       |  def foo(aaa: Int, bbb: Int, ccc: Int) = aaa + bbb + ccc
+       |  val k = foo (
+       |    bbb = 123,
+       |    aa@@
+       |  )
+       |} 
+       |""".stripMargin,
+    """|aaa = : Int
+       |""".stripMargin,
+    topLines = Some(1),
+  )
+
+  check(
+    "second-first2",
+    """|object Main {
+       |  def foo(aaa: Int, bbb: Int, ccc: Int) = aaa + bbb + ccc
+       |  val k = foo (
+       |    bbb = 123,
+       |    ccc = 123,
+       |    aa@@
+       |  )
+       |} 
+       |""".stripMargin,
+    """|aaa = : Int
+       |""".stripMargin,
+    topLines = Some(1),
+  )
+
+  check(
+    "second-first3",
+    """|object Main {
+       |  def foo(ddd: Int)(aaa: Int, bbb: Int, ccc: Int) = aaa + bbb + ccc
+       |  val k = foo(123)(
+       |    bbb = 123,
+       |    ccc = 123,
+       |    aa@@
+       |  )
+       |} 
+       |""".stripMargin,
+    """|aaa = : Int
+       |""".stripMargin,
+    topLines = Some(1),
+  )
+
+  check(
+    "second-first4".tag(IgnoreScala2),
```

Comment:
Should we raise a separate issue?

Reply:
Because of `IgnoreScala2`? It's mostly because we don't have named arg completions for Scala 2 at all

Reply:
ach ok, I think that is fine